src/numberButtons.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function NumberButtons({ onClick }) {
+  return (
+    <p>
+      {[1, 2, 3, 4, 5].map((i) => <button type="button" key={i} onClick={() => onClick(i)}>{i}</button>)}
```

Comment:
```suggestion
      {[1, 2, 3, 4, 5].map((i) => (
        <button 
          type="button"
          key={i}
          onClick={() => onClick(i)}
        >
            {i}
        </button>
      )}
```

괄호가 하나 빠졌어요!

그리고 `props` 가 많을 때 저는 저렇게 formatting  하는편인데 어떤게 더 가독성이 높은 것 같나요?

Reply:
오 정말 빠졌네요!! formatting을 수정했습니다!